packages/server/src/cli.ts

```diff
@@ -166,6 +166,26 @@ if (isCli) {
 		password = buffer.toString("hex");
 	}
 
+	// If CLI password was provided, obsfucate password from process title
+	if (options.password) {
+		const parts = [process.title];
+		parts.push(path.relative(process.cwd(), process.argv[1]));
```

Comment:
When running a script with `node` this is correct but when running the binary this won't normally be present in the process listing, right? I think you'll get something like `./code-server out/cli.js --password ****` instead of just `./code-server --password ****`. (It'll also be truncated since the title cannot increase in length.)